### PR TITLE
Update data-privacy.md

### PR DESCRIPTION
**Pull request by tymconn**
As per CELA Guidance from Jim Sfekas, page was updated to removed previously necessary customer consent language and replace with language around the data storage location of Microsoft Threat Protection data.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/mtp/data-privacy.md [hunk 1]
```diff
@@ -26,11 +26,9 @@ search.appverid:
 **Applies to:**
 - Microsoft Threat Protection
 
+Microsoft Threat Protection operates in Microsoft Azure data centers in the European Union, The United Kingdom, and the United States. Customer data collected by the service is stored at rest in (a) the geographic location of the tenant as identified during provisoning or, (b) if Microsoft Threat Protection uses another Microsoft online service to process such data, the geolocation as defined by the data storage rules of that other online service.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/data-privacy.md]:
```suggestion
Microsoft Threat Protection operates in Microsoft Azure data centers in the European Union, The United Kingdom, and the United States. Customer data collected by the service is stored at rest in (a) the geographic location of the tenant as identified during provisioning or, (b) if Microsoft Threat Protection uses another Microsoft online service to process such data, the geolocation as defined by the data storage rules of that other online service.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@mjcaparas - PR has been copy edited and is ready for final review.  

Thank you.

cc: @kenwith

**Comment by tymconn** [comment]
Hello All. What can be done to push this through today?

**Comment by UserAdminDEUS** [comment]
I did not do this to my phone y'all altered my data so it's more like y'all
are threatening my security because I cannot update my phone

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**